code/datums/extensions/assembly/assembly_interaction.dm

```diff
@@ -10,20 +10,9 @@
 		return TRUE
 
 	if(isWelder(W))
```

Comment:
Checks like these are also extraneous now, handled in `do_tool_interaction()`

Reply:
I've maintained the checks just for sanity's sake. I'm not sure if the do_tool_interaction() proc is solid enough to handle all use cases where is(x) calls would be.

Reply:
It is. The check is superseded by the tool interaction proc.